README.md

```diff
@@ -145,7 +145,7 @@ Checkout the [sample .env file](https://github.com/openlawteam/tribute-contracts
 
 **Required env vars per deployment type**
 
-- Ganache deployment: `DAO_NAME`, `ERC20_TOKEN_NAME`, `ERC20_TOKEN_SYMBOL`, `ERC20_TOKEN_DECIMALS`, `COUPON_CREATOR_ADDR`.
+- Ganache deployment: `DAO_NAME`, `DAO_OWNER_ADDR`, `ERC20_TOKEN_NAME`, `ERC20_TOKEN_SYMBOL`, `ERC20_TOKEN_DECIMALS`, `COUPON_CREATOR_ADDR`.
```

Comment:
When I tried to deploy to `ganache` without this set, I got an invalid argument error for the `owner` argument being empty [here](https://github.com/openlawteam/tribute-contracts/blob/master/utils/DeploymentUtil.js#L65). I may have been doing something wrong, happy to remove / update other documentation if so.

Reply:
good catch @jessepollak. That's correct.